### ISSUE TITLE
workspace `@dashevo/drive` does not exist

### ISSUE THREAD

**Issue by coolaj86** [dashpay/platform#1534]
```sh
yarn workspaces focus --production @dashevo/drive
```

```text
Internal Error: Workspace not found (@dashevo/drive)
```

Working from https://docs.dash.org/en/stable/docs/user/masternodes/setup-testnet.html#drive

**Comment by pshenmic** [comment]
Yes, you are correct. Docs have not been updated after migration Drive to Rust

**Comment by coolaj86** [comment]
@pshenmic How do we get the official docs updated with this info? Is there a repo for it somewhere?

**Comment by thephez** [comment]
For pages on docs.dash.org, you can click the page's "Edit on GitHub" link in the right sidebar. That will take you to the correct file/repo.
